lib/tasks/csv-import.rake

```diff
@@ -0,0 +1,39 @@
+require 'csv'
+
+namespace "csv" do
+  desc "Import a CSV file with recommended links into the search-admin database"
+  task "import" => :environment do
+    filename = ENV["filename"]
+    links = []
+
+    # Check the CSV file exists
+    fail "The file #{filename} does not exist." unless File.exist?(filename)
+
+    CSV.foreach(filename, encoding: "UTF-8", headers: true) do |row|
+      next if row.empty?
+
+      # Create a new recommended link
+      puts "Creating recommended link for #{row[0]}"
+      link = RecommendedLink.new(
+        title: row[0],
+        link: row[1],
+        description: row[2],
+        keywords: row[3],
+        search_index: row[4]
+      )
+
+      # Check the recommended link is valid
+      if link.valid?
+        links.push link
+      else
+        fail "Error(s) on line #{$.} of the CSV file: #{link.errors.messages}"
+      end
+    end
+
+    # Save all recommended links to the database
```

Comment:
I feel that all the comments in this task are redundant. The code is clear enough.